Src/coffeescript/cql-execution/package.json

```diff
@@ -36,9 +36,12 @@
     "coffee-script": "^1.12.0",
     "coveralls": "^2.13.1",
     "mocha": "^3.4.2",
+    "nyc": "^11.4.1",
     "should": "^11.2.1"
   },
   "dependencies": {
+    "istanbul": "^0.4.5",
```

Comment:
Why is `istanbul` a normal dependency rather than a `devDependency`? Isn't it only needed for collecting coverage on test runs?